plugins/main/public/components/common/wazuh-discover/wz-flyout-discover.tsx

```diff
@@ -274,13 +275,14 @@ const WazuhFlyoutDiscoverComponent = (props: WazuhDiscoverProps) => {
           {isDataSourceLoading ? (
             <LoadingSpinner />
           ) : (
-            <WzSearchBar
-              appName='wazuh-discover-search-bar'
-              {...searchBarProps}
-              useDefaultBehaviors={false}
-              hideFixedFilters
-            />
-          )}
+              <WzSearchBar
+                appName='wazuh-discover-search-bar'
+                {...searchBarProps}
+                fixedFilters={fixedFilters}
+                useDefaultBehaviors={false}
+                hideFixedFilters
```

Comment:
**thought**: if the fixed filters go to be hidden, are they required by the component in this case?